src/main/resources/permissions.txt

```diff
@@ -101,3 +99,6 @@ instance-authority-links.instances.collection.get
 instance-authority-links.instances.collection.put
 instance-authority.linking-rules.collection.get
 user-tenants.collection.get
+organizations.organizations.collection.get
+invoices.acquisitions-units-assignments.assign
```

Comment:
it seems that for the data-import system user we should not add this permission as well because data import processes initiated by all other users will be provided with this permission, regardless of whether a user starting the import process has this permission or not.

Reply:
I'm not sure what to do, then, since the data import system user operates on the user's behalf to process jobs. Anything the user can do the system user must be able to do

Reply:
I haven't come to some solution, but might be we can consider approach of saving "X-Okapi-Permissions" header of the initial user along with the DataImportQueueItem (or smth like that) in the DB. This way, the header could be passed further through Kafka headers and be used by the logic that determines whether the user has this specific permission during invoice creation.